build.py

```diff
@@ -7,25 +8,35 @@
 from jinja2 import Environment, select_autoescape, FileSystemLoader
 
 from pipeline.translator import PythonBuilder
-from pipeline.utils import clone_sources, SchemaLoader
+from pipeline.utils import clone_sources, SchemaLoader, InstanceLoader
 
-print("***************************************")
+print("*********************************************************")
 print(f"Triggering the generation of Python package for openMINDS")
-print("***************************************")
+print("*********************************************************")
 
 # Step 1 - clone central repository in main branch to get the latest sources
-clone_sources()
+###clone_sources()
```

Comment:
this needs to go back no?

Reply:
and extended to clone schemas as well as instances?

Reply:
> this needs to go back no?

whoops! fixed